h/api/views.py

```diff
@@ -184,7 +185,9 @@ def create(request):
     return AnnotationJSONPresenter(request, legacy_annotation).asdict()
 
 
-@api_config(route_name='api.annotation', request_method='GET', permission='read')
+@api_config(route_name='api.annotation',
```

Comment:
It's really helpful when reviewing if semantic and non-semantic changes aren't bundled up in the same diff. Reformatting code to fit in 78 characters while making meaningful changes to the same code makes it much more likely that things are going to be missed, so let's not do this. Not least when we have the line-length rules disabled in our setup.cfg anyway because they're stupid.